recipes/dsrc/meta.yaml

```diff
@@ -1,5 +1,5 @@
 package:
-  name: dscr
+  name: dsrc
   version: 2014.12.17
 
```

Comment:
Hi, I've merged the pull request unfortunately. So, I'll create another one with the md5 is that's okay (instead of sha256)